crates/ingress-kafka/src/subscription_controller.rs

```diff
@@ -129,9 +129,7 @@ impl Service {
             .with_context(|| format!("KafkaOptions is expected to contain the cluster '{}'. This might happen if you registered a subscription with a cluster name, but this cluster is not available anymore in the configuration. Configured Kafka clusters: {:?}", cluster, options.available_kafka_clusters()))?;
 
         client_config.set("metadata.broker.list", cluster_options.brokers.join(","));
-        for (k, v) in cluster_options.additional_options.clone() {
-            client_config.set(k, v);
-        }
```

Comment:
Why is it ok to not transfer the additional options from `cluster_options` to `client_config`?